spec/models/candidate_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe Candidate do
+  it "has a RETURNER constant" do
```

Comment:
I don't think these specs are adding any value - it should fail elsewhere if these don't exist

Reply:
Tell Sonarcloud

Reply:
If Sonarcloud is complaining about coverage of these then you're missing test coverage elsewhere